webapp/src/Controller/Team/SubmissionController.php

```diff
@@ -236,6 +236,10 @@ public function downloadAction(int $submitId): Response
                 $submitId));
         }
 
+        if ($this->submissionService->getSubmissionFileCount($submission) == 1) {
```

Comment:
```suggestion
        if ($this->submissionService->getSubmissionFileCount($submission) === 1) {
```

If we can be very exact I prefer that. It helps steering PHPStan to better results.